devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -203,7 +181,7 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
-    if (cmd.value == 0 && device.latestState("color").value != "#ffffff") {
+    if (device.latestState("color") == null || (cmd.value == 0 && device.latestState("color").value != "#ffffff")) {
         sendEvent(name: "color", value: "#ffffff", displayed: true)
```

Comment:
If it gets an off from us, or someone unplugs it, it defaults to white on the next on. It is really meant to be a night-light or a status light, which is why I am not being super picky about it maintaining state.